dev/com.ibm.ws.logging.osgi/resources/com/ibm/ws/logging/internal/osgi/resources/OSGiMessages.nlsprops

```diff
@@ -30,15 +30,19 @@
 # -------------------------------------------------------------------------------------------------
 
 # OSGI Log Message
-OSGI_MSG001={0} {1} {2} {3} {4}
+OSGI_MSG001=CWWKE0703I: {0} {1} {2} {3}
 OSGI_MSG001.explanation=An informational message was logged using the OSGi log service
 OSGI_MSG001.useraction=No action is required.
 
-OSGI_WARNING_MSG=CWWKE0700W: {0} {1} {2} {3} {4}
+OSGI_AUDIT_MSG=CWWKE0704A: {0} {1} {2} {3}
+OSGI_AUDIT_MSG.explanation=An audit message was logged using the OSGi log service
```

Comment:
Add a period at the end of the sentence.

Reply:
I'll fix that in a follow on commit.  Looks like I need to put a period at the end of the existing OSGI_MSG001.explanation text also (which is where I copied this message from.

Reply:
This is now fixed with PR #6119